roles/tile.rb

```diff
@@ -37,16 +37,17 @@
     :settings => {
       :defaults => {
         :max_connections => "250",
-        :temp_buffers => "32MB",
+        :shared_buffers => "16GB",
         :work_mem => "128MB",
+        :maintenance_work_mem => "8GB",
```

Comment:
This is also currently configured on a per-server basis.

Reply:
We've got cases where it's configured both on a role level and a server level. I'll remove the setting of it for many of the servers where the defaults make sense.